src/lib/logical_query_plan/mock_node.cpp

```diff
@@ -24,7 +24,7 @@ MockNode::MockNode(const std::shared_ptr<TableStatistics>& statistics) : Abstrac
 
 std::shared_ptr<AbstractLQPNode> MockNode::_deep_copy_impl() const {
   Fail("Cannot deep_copy MockNodes because we cannot get a deep copy of the statistics");
-  return nullptr;
+  // return nullptr;
```

Comment:
Why didn’t remove this line here as everywhere else?

Reply:
This was my test line. Thanks for pointing it out